src/base/components/Dashboard.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { connect } from "react-redux";
+
+class DashboardPage extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      sensorData: []
+    }
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
```

Comment:
what this? we can't rely on load timing.

Reply:
Sorry forgot to remove

Reply:
not using this

Reply:
removed!